packages/ui-kit-stage/HeaderBar.tsx

```diff
@@ -0,0 +1,23 @@
+/**
+ * Component: HeaderBar
+ * Added: 2018-07-08
+ * JIRA: https://jira.mesosphere.com/browse/DCOS-39074
+ */
+import * as React from "react";
+import { coreColors } from "@dcos/ui-kit/dist/packages/shared/styles/color";
+
+const headerBarStyles: object = {
```

Comment:
if it's a UI Kit candidate why not using emotion here? Example: https://github.com/dcos-labs/ui-kit/blob/master/packages/badge/components/badge.tsx

Reply:
IMHO that would be part of the ui-kit issue, there is no emotion in dcos-ui yet 🤷‍♂️ If I finalize the component here, then there is no need for this staging folder thing… the Idea was to move on fast and optimize later instead of blocking this whole PR until that component is finalized 😕

Reply:
@juliangieseke @nLight I would not suggest adding this in the UI Kit. This case is more structural than functional/composable.

it's just returning a div with styles. what should be happening is the components inside this div are part of the UI Kit, not the div wrapper part of the UI Kit.